react/src/components/modals/SprkModal.js

```diff
@@ -260,42 +256,38 @@ class SprkModal extends Component {
               )}
             </header>
 
-            <div>
-              <div
-                className="
-                  sprk-o-Stack__item
-                  sprk-c-Modal__body
-                  sprk-o-Stack
-                  sprk-o-Stack--medium
-                "
-                id={'modalContent_' + this.aria_id}
-              >
-                {isWait && (
-                  <SprkSpinner
-                    size="large"
-                    additionalClasses="
+            <div
+              className={classnames(
+                'sprk-o-Stack__item sprk-c-Modal__body',
+                isWait ? 'sprk-o-Stack sprk-o-Stack--medium' : '',
```

Comment:
This needs to be updated to the below:

`{ 'sprk-o-Stack sprk-o-Stack--medium': isWait }`